kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/service/impl/ElectronicFilingServiceImpl.java

```diff
@@ -0,0 +1,169 @@
+package edu.arizona.kfs.module.purap.document.service.impl;
+
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+import org.kuali.rice.krad.service.BusinessObjectService;
+
+import edu.arizona.kfs.module.purap.document.service.ElectronicFilingService;
+import edu.arizona.kfs.module.purap.service.TaxReporting1099Service;
+import edu.arizona.kfs.tax.TaxConstants;
+import edu.arizona.kfs.tax.businessobject.ElectronicFile;
+import edu.arizona.kfs.tax.businessobject.Payee;
+import edu.arizona.kfs.tax.businessobject.PayeeRecord;
+import edu.arizona.kfs.tax.businessobject.Payer;
+import edu.arizona.kfs.tax.businessobject.RecordGroup;
+import edu.arizona.kfs.tax.service.TaxParameterHelperService;
+import edu.arizona.kfs.tax.util.RecordUtil;
+
+public class ElectronicFilingServiceImpl implements ElectronicFilingService {
+
+	private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ElectronicFilingService.class);
+
+	private BusinessObjectService businessObjectService;
+	private TaxReporting1099Service taxReporting1099Service;
+	private ParameterService parameterService;
+	private TaxParameterHelperService taxParameterHelperService;
+
+	private static final String NINE_SPACES = "         ";
+
+	public BusinessObjectService getBusinessObjectService() {
+		if (businessObjectService == null) {
+			businessObjectService = SpringContext.getBean(BusinessObjectService.class);
+		}
+		return businessObjectService;
+	}
+

```

Comment:
Fixed. In making the change to Spring Injection, it seems that 2 helper objects are unused. They have been removed.